src/main/java/edu/ksu/canvas/attendance/controller/AttendanceSummaryController.java

```diff
@@ -19,7 +19,10 @@
 import org.springframework.web.servlet.ModelAndView;
 
 import javax.servlet.ServletOutputStream;
+import javax.servlet.ServletRequest;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.servlet.ServletRequest'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AUselessImportCheck)